R/zzz.R

```diff
@@ -393,8 +404,7 @@ UpdateManifest <- function() {
       }
     }
     # Convert each entry to a dataframe and bind everything together
-    avail.pkgs <- lapply(X = avail.pkgs, FUN = as.data.frame, stringsAsFactors = FALSE)
-    avail.pkgs <- do.call(what = 'rbind', args = avail.pkgs)
+    avail.pkgs <- as.data.frame(t(avail.pkgs))
```

Comment:
This is not related to "no Description data" case but I need to this change to make `devtools::install_github()` workable.